app/src/main/java/com/infomaniak/mail/workers/BaseCoroutineWorker.kt

```diff
@@ -54,5 +54,7 @@ abstract class BaseCoroutineWorker(appContext: Context, params: WorkerParameters
 
     companion object {
         private const val MAX_RETRIES = 3
+
+        fun Data.getLongOrNull(key: String) = getLong(key, 0L).run { if (this == 0L) null else this }
```

Comment:
This feels like a strange place to keep this method in. It's never used in this file. It looks more like some kind of extension util